lib/preprocess.js

```diff
@@ -143,13 +143,18 @@ export const fetchMeasurements = async cid => {
       for await (const buf of entry.content()) {
         bufs.push(buf)
       }
-      const json = Buffer.concat(bufs).toString()
-      return JSON.parse(json)
+      return parseMeasurements(Buffer.concat(bufs).toString())
     }
   }
   throw new Error('No measurements found')
 }
 
+export const parseMeasurements = str => {
+  const ret = str.split('\n').filter(Boolean).map(line => JSON.parse(line))
+  if (Array.isArray(ret[0])) return ret[0]
```

Comment:
Should we make this check more strict?

```suggestion
  if (ret.length === 1 && Array.isArray(ret[0])) return ret[0]
```